yarn.lock

```diff
@@ -1381,11 +1381,6 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
-"@babel/standalone@^7.4.5":
```

Comment:
Wondering why there are lock changes when there are not package.json changes?

Reply:
this is a package.json change. I removed `"@storybook/addon-info": "^5.2.1"` because we aren't using it in v3.

Reply:
Oh sorry, I must have missed it. Space brain.